app/src/test/java/com/example/partyPlayer/UnitTest.kt

```diff
@@ -1,15 +1,14 @@
-package com.example.party_player
+package com.example.partyPlayer
 
 import org.junit.Test
 
 import org.junit.Assert.*
 
 /**
  * Example local unit test, which will execute on the development machine (host).
- *
  * See [testing documentation](http://d.android.com/tools/testing).
  */
-class ExampleUnitTest {
+class UnitTest {
```

Comment:
Changed the name of the test files to be a little more visibly unique. (Both test files began with "Example")

Reply:
While you're removing the "Example" prefix, rename the test to something that is more descriptive of what is being tested.  "UnitTest" is implied here, based on the location of this class.  What is being unit tested?  Put that in the name of the class.